types/esri-leaflet/index.d.ts

```diff
@@ -2,10 +2,11 @@
 // Project: http://esri.github.io/esri-leaflet
 // Definitions by: strajuser <https://github.com/strajuser>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.3
 
-/// <reference types="leaflet" />
+import * as L from 'leaflet';
 
-declare namespace L {
```

Comment:
@DenisCarriere it has to be that way now that the UMD format is used in the leaflet typing. It's very unfortunate because it meant I had to update all of the libraries depending on it. Trying to use `declare namespace L` now results in an error stating it can't be found. Typescript docs:

General library shape typing docs: https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html
UMD typing example: https://www.typescriptlang.org/docs/handbook/declaration-files/templates/module-d-ts.html
UMD module/extension example: https://www.typescriptlang.org/docs/handbook/declaration-files/templates/module-plugin-d-ts.html

Reply:
👍 Oh perfect, I wasn't aware of this Leaflet change, this PR makes sense now.

Leaflet has always been a special one with `declare namespace L`.

@alejo90 Thanks for the references!